code/modules/xenoarchaeology/traits/xenoartifact_majors.dm

```diff
@@ -0,0 +1,589 @@
+//Major traits - The artifact's main gimmick, how it interacts with the world
+///============
+/// Capture, moves target inside the artifact
+///============
+/datum/xenoartifact_trait/major/capture
+	desc = "Hollow"
+	label_desc = "Hollow: The shape is hollow, however the inside is deceptively large."
+	flags = BLUESPACE_TRAIT | PLASMA_TRAIT | URANIUM_TRAIT
+	var/spawn_russian = FALSE
+
+/datum/xenoartifact_trait/major/capture/on_init(obj/item/xenoartifact/X)
+	if(prob(1))
+		spawn_russian = TRUE
+
+/datum/xenoartifact_trait/major/capture/activate(obj/item/xenoartifact/X, atom/target)
+	if(isliving(X.loc))
+		var/mob/living/holder = X.loc
+		holder.dropItemToGround(X, thrown = TRUE)
+	if(ismovable(target) && !(istype(target, /obj/structure)) || !(istype(target, /obj/machinery)))
```

Comment:
This also needs to blacklist effects, docking port structures and a bunch of other shit.
Whitelist this to items and mobs rather than blacklisting, since it works better in the future